massdriver-application-helm/main.tf

```diff
@@ -1,13 +1,12 @@
 
 module "application" {
-  source  = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=fc5f7b1"
```

Comment:
This and the change on line 10 are paired and rely on this change from 2 weeks ago: https://github.com/massdriver-cloud/terraform-modules/commit/87cc8c237797bb0249140d2e6ba7cbe04822df56

`oidc_issuer_url` is a duplicate field (the value is in the `cluster_artifact` which is already being passed). Bumping the module ref here to get the changes.